### DISCUSSION TITLE
Image size that can be processed

### DISCUSSION THREAD

**Discussion by mskey17** [ultralytics/yolov5#9147] [Q&A]
I want to process the h:20863,w30637 image with detect.py, but it doesn't output properly.
When I debug it, in the for statement of yolo.py >> Model >> def _forword_once
x = m(x) # run
![ss](https://user-images.githubusercontent.com/60923907/186603819-ae93a419-a3ec-4d07-8fc4-7a461132bcad.png)

However, after repeating this several times, the program terminates. Do you know the reason for this?

・The process completed normally if the option was set to --imgsz 10431 15318.
・As long as you check with the task manager, the CPU capacity seems to be fine.
・If there is a limit to the size of the processed image, please tell me the threshold.

**Comment by spacewalk01** [comment]
Probably memory overflow problem

**Reply by spacewalk01** [reply]
Why not split it into patches and perform detection for each patch sequentially and stitch outputs together
